packages/visual-editor/src/components/pageSections/ReviewsSection.tsx

```diff
@@ -21,16 +24,125 @@ const ReviewsSectionInternal: React.FC<ReviewsSectionProps> = (
   props: ReviewsSectionProps
 ) => {
   const { t } = useTranslation();
+  const [totalReviews, setTotalReviews] = React.useState(0);
+  const [averageRating, setAverageRating] = React.useState(0);
+  React.useEffect(() => {
+    fetchReviews()
+      .then((reviews) => {
+        if (!reviews || !reviews.meta) {
+          throw new Error("Invalid response structure from API");
+        }
+        if (reviews?.meta?.errors && reviews.meta.errors.length > 0) {
+          throw new Error(
```

Comment:
I'm not sure you want to throw in the component because the api returns an error. I would console error the actual error and then show no reviews.